h5p-row.js

```diff
@@ -94,7 +94,53 @@ H5P.Row = (function (EventDispatcher) {
       $container.addClass('h5p-row').html('').append(wrapper);
     };
 
-    // TODO: Add required xAPI stuff once we get child instances
+    /**
+     * Get xAPI data.
+     * Contract used by report rendering engine.
+     *
+     * NB! This implementation is different from expected use of getXAPIData,
+     * as row is only used as a subcontent container, and would therefore
```

Comment:
@VildeStabell So, does this mean that any software that tries to make sense of H5P's xAPI data will have to add exceptions for Row?

Reply:
@otacke That shouldn't be necessary no. Row is only meant as a subcontent to column, which has specific handling for Row when it comes to xAPI. Outside of column, the xAPI data should look the same as a Column without Row. Using the getXAPIData function was convenient, but if you forsee problems despite the handling in Column I could move it somewhere else.

Reply:
I have not looked at any of the code. I just get suspicious about any hardcoded exceptions in code anywhere, in particular if it touches an API (contract) function that others may use and rely on to show a consistent behavior.
There's the reverse case in Single Choice Set, kind of, sending out (inconsistent) xAPI data for a virtual subcontent instance that caused extra work for a client before.

Reply:
Hm, good point. I'll look into moving it into a non-contract function, just in case